app/common/console_output.py

```diff
@@ -0,0 +1,102 @@
+from app.common.console_output_segment import ConsoleOutputSegment
+
+
+class ConsoleOutput(object):
+    """
+    Represents the console output of an atom.
+    """
+    def __init__(self, path):
+        """
+        :param path: sys path to the console output file
+        :type path: str
+        """
+        self.path = path
+
+    def segment(self, max_lines=50, offset_line=None):
+        """
+        Return a segment of the console output.
+
+        :param max_lines:
```

Comment:
It's pretty obvious what this is supposed to be but did you mean to add a docstring for the param?